tests/utils.py

```diff
@@ -27,17 +34,46 @@ def login(flask_client) -> User:
     return user
 
 
-def create_user(flask_client) -> User:
-    # create user, user is activated
+def random_token(length: int = 10) -> str:
+    return "".join(random.choices(string.ascii_lowercase + string.digits, k=length))
+
+
+def create_random_user() -> User:
+    email = "{}@{}.com".format(random_token(), random_token())
     return User.create(
-        email="a@b.c",
+        email=email,
         password="password",
         name="Test User",
         activated=True,
         commit=True,
     )
 
 
+def create_random_alias(user: User) -> Alias:
+    alias_email = "{}@{}.com".format(random_token(), random_token())
+    alias = Alias.create(
+        user_id=user.id,
+        email=alias_email,
+        mailbox_id=user.default_mailbox_id,
+        commit=True,
+    )
+    return alias
+
+
 def pretty(d):
     """pretty print as json"""
     print(json.dumps(d, indent=2))
+
+
+def load_eml_file(
+    filename: str, template_values: Optional[Dict[str, str]] = None
+) -> EmailMessage:
+    emails_dir = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "example_emls"
+    )
+    fullpath = os.path.join(emails_dir, filename)
+    template = jinja2.Template(open(fullpath).read())
```

Comment:
should we use the `with open()` pattern here to close the file automatically?

Reply:
It doesnt' change much for read only files. the file will be closed when the handle is dropped. For write files is more important because we'd might need to ensure the file is saved before doing other stuff. 
I'll change it to use with.